Source/Components/TrackManagerUI.cpp

```
@@ -368,30 +368,71 @@ void TrackManagerUI::renderToolbar(AestraUI::NUIRenderer& renderer) {
-    const auto idleBg = themeManager.getColor("buttonBgDefault").withAlpha(0.98f);
-    const auto hoverBg = themeManager.getColor("buttonBgHover").withAlpha(0.99f);
+    const auto idleBg = themeManager.getColor("buttonBgDefault").withAlpha(0.94f);
+    const auto hoverBg = themeManager.getColor("buttonBgHover").withAlpha(0.97f);
     const auto activeBg = themeManager.getColor("buttonBgActive").withAlpha(0.99f);
-    const auto idleBorder = themeManager.getColor("border").withAlpha(0.28f);
+    const auto idleBorder = themeManager.getColor("border").withAlpha(0.24f);
     const auto hoverBorder = themeManager.getColor("border").withAlpha(0.40f);
     const auto activeBorder = themeManager.getColor("borderActive").withAlpha(0.24f);
     const auto innerHighlight = AestraUI::NUIColor::white().withAlpha(0.025f);
     const auto shadowColor = AestraUI::NUIColor(0, 0, 0, 0.14f);
+    const auto utilityBg = themeManager.getColor("surfaceRaised").withAlpha(0.58f);
```

Cppcheck (2.20.0)

[style] The function 'decrement' is never used.

(unusedFunction)